### DISCUSSION TITLE
bugs during running the incompressible optimization  for the airfoil

### DISCUSSION THREAD

**Discussion by MachAeroDeveloper** [mdolab/dafoam#582] [General]
h ave two problem professor 
1- i use the same runscript.py for naca0012 airfoil  after i chnage the airfoil of s809 and everything is working which i have been changed the coordinate of airfoil and the ffd point and the mesh and every thing working fine till i get this error i hope you help me on this problem and a appreciate your reply and help and thank you advanced 
2- the file paraview.foam is empty and can not give any result of my optimization 
![image](https://github.com/mdolab/dafoam/assets/157682082/753b2346-a69f-4fac-b1d9-b1669ea3f163)

**Comment by friedenhe** [comment]
It seems that your case converged without an error. The message you saw is a warning. ParaView.foam is a dummy file. Please check the DAFoam workshop slides and videos to get the basics of post-processing.

**Reply by MachAeroDeveloper** [reply]
thank you so much , i copy the folder to the windows and everything is working fine ,but the paraview on the linex ubunto not working and no show any result .

**Comment by friedenhe** [comment]
Please follow "Get Started" section from https://dafoam.github.io to do post-processing.
